tests/AppBundle/Controller/Admin/BlogControllerTest.php

```diff
@@ -84,9 +84,9 @@ public function testBackendHomepage()
 
         $crawler = $client->request('GET', '/en/admin/post/');
 
-        $this->assertCount(
-            30,
-            $crawler->filter('body#admin_post_index #main tbody tr'),
+        $this->assertGreaterThanOrEqual(
+            1,
+            $crawler->filter('body#admin_post_index #main tbody tr')->count(),
```

Comment:
Since authors are randomly assigned, each time the fixtures are loaded this test would fail (depending on the authenticated user)